cmake/modules/AddSwift.cmake

```diff
@@ -310,6 +310,7 @@ function(_add_variant_c_compile_flags)
       list(APPEND result "\"${CMAKE_INCLUDE_FLAG_C}${path}\"")
     endforeach()
     list(APPEND result "-D__ANDROID_API__=${SWIFT_ANDROID_API_LEVEL}")
+    list(APPEND result -funwind-tables)
```

Comment:
We should do this for Linux as well.

Reply:
Seems that at least my Linux worked OK without this. Should I add it there, even if it already works? Should I add it for every architecture or only for ARMs?

Reply:
I think that adding it for ARM only is the best idea.  x86/x86_64/ppc/ppc64 use alternate mechanisms for unwinding, and I believe that those are the only other Linux platforms we support.  AArch64 has a frame pointer IIRC, and is not entirely dependent on the `.arm.exidx` and `.arm.extab` sections constructed by `-funwind-tables` on ARM EABI.